sql/src/test/java/io/crate/analyze/SelectStatementAnalyzerTest.java

```diff
@@ -1910,19 +1910,12 @@ public void testSelectStarFromUnnest() throws Exception {
 
     @Test
     public void testSelectStarFromUnnestWithInvalidArguments() throws Exception {
-        expectedException.expect(IllegalArgumentException.class);
-        expectedException.expectMessage("unnest expects arguments of type array. Got an argument of type 'long' at position 0 instead.");
+        expectedException.expect(UnsupportedOperationException.class);
+        expectedException.expectMessage("unknown function: unnest(long, string)");
         analyze("select * from unnest(1, 'foo')");
     }
 
     @Test
-    public void testSelectStarFromUnnestWithNoArguments() throws Exception {
```

Comment:
because this case is now covered also by the other argument validation test, since it is no special case anymore to have no arguments